src/chrome.configuration-store.spec.ts

```diff
@@ -30,10 +35,48 @@ describe('ChromeStore', () => {
   });
 
   afterEach(() => {
+    jest.useRealTimers();
     jest.clearAllMocks();
   });
 
-  it('is always expired', async () => {
+  it('is always expired without cooldown', async () => {
+    chromeStore = new ChromeStorageAsyncStore(extendedStorageLocal, undefined);
+    expect(await chromeStore.isExpired()).toBe(true);
+  });
+
+  it('is not expired with cooldown', async () => {
+    chromeStore = new ChromeStorageAsyncStore(extendedStorageLocal, 10);
+
+    (extendedStorageLocal.get as jest.Mock).mockImplementation(() => {
+      return Promise.resolve({
+        ['eppo-configuration']: JSON.stringify(mockEntries),
+        ['eppo-configuration-meta']: JSON.stringify({
+          lastUpdatedAtMs: new Date().getTime(),
+        }),
+      });
+    });
+
+    expect(await chromeStore.isExpired()).toBe(false);
+  });
+
+  it('is expired after cooldown', async () => {
+    chromeStore = new ChromeStorageAsyncStore(extendedStorageLocal, 10);
+
+    (extendedStorageLocal.get as jest.Mock).mockImplementation(() => {
+      return Promise.resolve({
+        ['eppo-configuration']: JSON.stringify(mockEntries),
+        ['eppo-configuration-meta']: JSON.stringify({
+          lastUpdatedAtMs: now,
+        }),
+      });
+    });
+
+    // advance time by 5 seconds
+    await jest.advanceTimersByTimeAsync(5 * 1000);
```

Comment:
Hello old friend 😅

Reply:
this really came in handy thank you for demonstrating how to do it - it feels very cool to be able to write a test like this.